README.md

```diff
@@ -16,7 +16,18 @@ global:
   imagePullSecrets:
     - registry
   secret:
-    dockerconfigjson: "BASE64_ENCODED_JSON_CONTAINING_TOKEN"
+    dockerconfigjson: "BASE64_ENCODED_JSON_CONTAINING_DOCKER_AUTH_CONFIG"
+```
+
+You can generate this value with a bash command like this
+```bash
+echo '{
+        "auths": {
+                "ghcr.io": {
+                        "auth": "TO_BE_REPLACED"
+                }
+        }
+}' | sed "s/TO_BE_REPLACED/$(echo "username:token" | base64 -w 0 )/g" | base64 -w 0
```

Comment:
... replace "username:token" with the credentials supplied by G DATA

or something like that to clarify?